packages/react-components/react-progress/src/stories/Progress/ProgressDefault.stories.tsx

```diff
@@ -2,13 +2,13 @@ import * as React from 'react';
 import { Progress, ProgressProps } from '@fluentui/react-progress';
 
 export const Default = (props: Partial<ProgressProps>) => {
-  return <Progress {...props} percentComplete={50} />;
+  return <Progress {...props} value={0.5} />;
 };
 
 Default.parameters = {
   docs: {
     description: {
-      story: `Default determinate Progress bar`,
+      story: `Determinate Progress bar`,
     },
   },
 };
```

Comment:
(nit) I'd leave the word "determinate" out of the documentation. Progress bars can either have a value, or be indeterminate.

In the case of the Default story, you can probably remove the `Default.parameters` entirely. It looks like many of our default stories don't have a description at all. It's covered by ProgressDescription.md